src/test/java/com/solvd/micro9/users/integration/TestcontainersTest.java

```diff
@@ -45,4 +48,12 @@ public Map<String, Object> getConsumerProps(final Class<?> clazz) {
         return props;
     }
 
+    public ObjectMapper getObjectMapper() {
```

Comment:
@eyakauleva move this logic to Spring Test configuration

Reply:
fixed 1219506